Crypto/Random.hs

```diff
@@ -49,7 +50,7 @@ seedLength = 40
 
 -- | Create a new Seed from system entropy
 seedNew :: MonadRandom randomly => randomly Seed
-seedNew = Seed `fmap` getRandomBytes seedLength
+seedNew = (Seed . B.take seedLength . B.convert . (hash :: ScrubbedBytes -> Digest SHA512)) `fmap` getRandomBytes 64
```

Comment:
As I said, please write a comment why hashing is necessary here.

Reply:
@epoberezkin would you be willing to do this? If not, I can do so. I'd essentially just copypaste your argument from #22